CHANGES.rst

```diff
@@ -15,6 +15,14 @@ New features:
 
 Bug fixes:
 
+- *add item here*
+
+
+1.2.12 (unreleased)
+-------------------
+
+Bug fixes:
+
```

Comment:
@petschki looks like the merge left some garbage behind - 2x unreleased?

Reply:
well, I'd suggest to release my fix in 1.2.12 and update the release date later for 1.3.0 changelog